sirepo/runner_api.py

```diff
@@ -23,6 +24,26 @@
 #: What is_running?
 _RUN_STATES = ('pending', 'running')
 
+
+@api_perm.require_user
+def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=None):
+#TODO(robnagler) validate suffix and frame
+    sim = http_request.parse_params(
+        id=simulation_id,
+        model=model,
+        type=simulation_type,
+    )
+    f, c, t = sirepo.template.import_module(sim.type).get_data_file(
+        simulation_db.simulation_run_dir(sim.req_data),
+        sim.sim_data.compute_model(sim.model),
+        int(frame),
+        options=sim.req_data.copy().update(suffix=suffix),
+    )
+    return http_reply.headers_for_no_cache(
+        http_reply.as_attachment(flask.make_response(c), t, f),
```

Comment:
This should be app.response_class, not flask.make_response. Encapsulate this complete in http_reply.